### PR TITLE
support __CONSTRUCTOR keyword and replace it with monitorName at comp…

### PR DESCRIPTION
**Pull request by yzhang90**
…ilation time.

### CODE REVIEW

File: rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java [hunk 1]
```diff
@@ -42,7 +42,9 @@ public RawMonitor(String outputName, RVMonitorSpec rvmSpec,
                     rvmSpec.getEvents(), coenableSet);
         }
 
-        monitorDeclaration = new UserJavaCode(rvmSpec.getDeclarationsStr());
+        String declStr = rvmSpec.getDeclarationsStr();
+        declStr = declStr.replaceAll("__CONSTRUCTOR", monitorName.toString());
```
Comment (owolabileg) [inline rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java:49]: Please document in comments what use case this is meant to support. Why do we need this?

**Review by owolabileg — CHANGES_REQUESTED** [review]

File: rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java (hunk 1 above)
Comment (yzhang90) [reply rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java:49]: done

Comment (owolabileg) [reply rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java:49]: LGTM. Thanks.

Comment (yzhang90) [reply rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java:49]: Can you approve the PR?

**Review by owolabileg — APPROVED** [review]

**Review by owolabileg — APPROVED** [review]

File: rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java (hunk 1 above)
Comment (owolabileg) [reply rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/monitor/RawMonitor.java:49]: Done.

**Review by virgil-serbanuta — APPROVED** [review]

**Pull request merged**